README.md

```diff
@@ -8,7 +8,7 @@
 ```
 <custom-element-demo>
   <template>
-    <script src="amp-viewer.js"></script>
+    <link rel="import" href="amp-viewer.html" async>
```

Comment:
Question: any reason why to change this to an import? This comment is used by webcomponents.org to have a inline demo (see https://www.webcomponents.org/publish)
This element is not published yet on webcomponents.org, so I'd either not modify this line or completely remove code from line 8 to 15. @frankiefu WDYT?

Reply:
Agree with @valdrinkoshi and let's remove it (line 8 to 15).